src/porepy/viz/exporter.py

```diff
@@ -2391,9 +2520,28 @@ def _write(
                 else:
                     raise ValueError("Data values have wrong dimension")
 
+        for field in fields_pt:
+            # Although technically possible, as implemented, field.values should never
+            # be None.
+            assert field.values is not None
+
+            # For each field create a sub-vector for each geometrically uniform group
+            # of cells
+            point_data[field.name] = list()
+
+            if field.values.ndim == 1:
+                point_data[field.name] = field.values
+            elif field.values.ndim == 2:
+                point_data[field.name] = field.values.T
+            else:
+                raise ValueError("Data values have wrong dimension")
+
```

Comment:
Isn't it possible to combine this with the treatment of cell_data? I am thinking of nesting the loop and use `for data, fields_ in zip([cell_data, point_data], [fields, fields_pt]):` modulo my bad naming. This would allow for slightly less code.